core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -167,6 +167,10 @@ private TableProperties() {}
       "write.parquet.bloom-filter-max-bytes";
   public static final int PARQUET_BLOOM_FILTER_MAX_BYTES_DEFAULT = 1024 * 1024;
 
+  public static final String PARQUET_BLOOM_FILTER_COLUMN_FPP_PREFIX =
+      "write.parquet.bloom-filter-fpp.column.";
+  public static final double PARQUET_BLOOM_FILTER_COLUMN_FPP_DEFAULT = 0.01;
```

Comment:
Does this value currently match the default behavior? In other words, will existing users be impacted?

Reply:
Parquet currently defaults to [0.01](https://github.com/apache/parquet-mr/blob/parquet-1.13.x/parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java#L61)

Reply:
I checked `ParquetProperties`. Should be all good.

Reply:
Can we add comments for `fpp` and a test as well?

Reply:
Have you checked `TestBloomFilter`?